examples/combined_ODNP.py

```diff
@@ -72,6 +72,7 @@
 T1_powers_dB = gen_powerlist(
     config_dict["max_power"], config_dict["num_T1s"], three_down=False
 )
+T1_powers_dB = np.floor(T1_powers_dB)
```

Comment:
this works specifically for this purpose, but it's better to set the `min_dBm_step` argument of `gen_powerlist`.  Here, you can just click on the function name, and see the docstring.

Reply:
note that you can also test that your solution works by just writing a small script (not committing to the repo -- just so you can see it works) that prints the result of `gen_powerlist`